rust/abacus-base/Cargo.toml

```diff
@@ -15,31 +15,31 @@ ethers = { git = "https://github.com/gakonst/ethers-rs", branch = "master" }
 thiserror = { version = "1.0.22", default-features = false }
 async-trait = { version = "0.1.42", default-features = false }
 futures-util = "0.3.12"
-color-eyre = "0.6.1"
-tracing = "0.1.22"
-tracing-futures = "0.2.4"
-tracing-subscriber = "0.2.15"
+eyre = "0.6"
+tracing = "0.1"
+tracing-futures = "0.2"
+tracing-subscriber = { version = "0.3", features = ["json"] }
 rocksdb = { git = "https://github.com/rust-rocksdb/rust-rocksdb" }
 mockall = "0.10.2"
 
 abacus-core = { path = "../abacus-core" }
 abacus-ethereum = { path = "../chains/abacus-ethereum"}
 abacus-test = { path = "../abacus-test" }
 paste = "1.0.5"
-tracing-error = "0.1.2"
+tracing-error = "0.2"
 
 prometheus = "0.12"
 
 warp = "0.3"
 
 # these versions are important!
```

Comment:
So I did read this warning, but as far as I can tell they just mean they need to get updated together, so if you know more please let me know.